IntegrationTests/PurchaseTester/PurchaseTester/SwiftPaywall.swift

```diff
@@ -363,7 +363,11 @@ class SwiftPaywall: UIViewController {
             ])
         
         // The offerings loading indicator
-        offeringLoadingIndicator = UIActivityIndicatorView(style: .medium)
+        if #available(iOS 13.0, *) {
+            offeringLoadingIndicator = UIActivityIndicatorView(style: .medium)
+        } else {
+            offeringLoadingIndicator = UIActivityIndicatorView(style: .white)
+        }
```

Comment:
this is on me, I realized yesterday that this won't work with older OS

Reply:
No worries! Soon we'll be able to go iOS 13 only if we want 😄